cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -165,7 +165,7 @@ pub struct ManifestProfile {
 pub struct ManifestOptions {
     /// A list of systems that each package is resolved for.
     #[serde(default)]
```

Comment:
suggestion: remove serde `default`.
`Option` is already "optional"
```suggestion
```

Reply:
Fixed